sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -709,9 +711,13 @@ public Key restoreKey(byte[] backup) {
      * call {@link KeyClient#getKey(KeyBase baseKey)} . This will return the {@link Key key} with key material included of its latest version.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listKeys}
      *
-     * @return A {@link List} containing {@link KeyBase key} of all the keys in the vault.
```

Comment:
Description is fine, But the header should stay too.
<p><strong>Code Samples to iterate keys by page</strong></p>